### PR TITLE
I5-7005,I5-6585 -- Re-implement informer whitelabel PR & i18n - create new i18n plugin/client-focused customization readme

### PR DESCRIPTION
**Pull request by holly-weisser**

### CODE REVIEW

File: locales/custom/en/common.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "entrinsik": "ACME",
+  "informer": "Reporting",
+  "entrinsik_informer": "ACME Reporting",
+  "informer4": "Legacy Reporting",
```
Comment (gmsewell6) [inline locales/custom/en/common.json]: the i4 key was removed from the core, might not want to mislead to its existence

File: README.md [hunk 2]
```diff
@@ -1,9 +1,65 @@
...
+## To replace Logo and Favicon
+1. Put the main logo and banner logo (for login page) into public/images
+2. Put custom login page in assets/login.html
+3. To customize the favicon, use https://realfavicongenerator.net/ and put it in public/images/favicon folder
+
+## To override various text and translations within the core Informer application
+Clients wanting to override (partially or wholly) any part of the core or plugin-added namespaces,
+or even want to introduce entirely new languages for their ENTIRE DEPLOYMENT can do so one of two ways:
+    
+1. For native environments, define a 'customLocalesDir' in config.json under i18n:
```
Comment (gmsewell6) [inline README.md]: property actually changed since this was initially written, so to separate -- `... define a custom 'localesDir' in ...`

**Review by gmsewell6 — CHANGES_REQUESTED** [review]

File: locales/custom/en/common.json (hunk 1 above)
Comment (holly-weisser) [reply locales/custom/en/common.json]: done

File: README.md (hunk 2 above)
Comment (holly-weisser) [reply README.md]: done

**Review by gmsewell6 — APPROVED** [review]

**Pull request merged**